From 32784b4b4b57f210bf0169bea46f44db28869ea7 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Mon, 27 Apr 2026 14:50:36 -0500
Subject: [PATCH] Allow absolute RPATH to be set on rviz2 executable

It would be better if this was a relative RPATH, but we're installing
the executable to two different locations so it will require a little
bit of refactoring.
---
 rpm/ros-lyrical-rviz2.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/ros-lyrical-rviz2.spec b/rpm/ros-lyrical-rviz2.spec
index 5d2a58811..b813b0089 100644
--- a/rpm/ros-lyrical-rviz2.spec
+++ b/rpm/ros-lyrical-rviz2.spec
@@ -14,6 +14,8 @@ Source0:        %{name}-%{version}.tar.gz
 BuildRequires:  bloom-rpm-macros
 BuildRequires:  cmake
 
+%global __brp_check_rpaths QA_RPATHS=0x0002 %{__brp_check_rpaths}
+
 %{?bloom_package}
 
 %description
-- 
2.43.0

